### DISCUSSION TITLE
Making Clipboard work with Tabby terminal

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [zellij-org/zellij#2052] [Q&A]
I've recently switched over from tmux (via byobu) to Zellij (can't recommend it enough).

On Ubuntu/Pop!_OS with their native/local terminals, the clipboard functionality works out-of-the-box very well. It seems Zellij is passing the proper control characters to communicate a clipboard operation.

I use [Tabby](https://github.com/Eugeny/tabby) on Windows to connect to those machines remotely, but using the clipboard isn't transferred.

NOTE: Clipboard works with the `[Shift]` key, but I'm looking for a way to make zellij communicate the clipboard so marking is confined to a pane.

As I'm not very familiar with the clipboard terminal communication, and was hoping someone here might have an idea how to implement a functional clipboard with Tabby and zellij.

I'm also cross-posting this question on Tabby's discussion board. [link](https://github.com/Eugeny/tabby/discussions/7716)

**Comment by LiuLinyun** [comment]
I have got the same problem. Because of this problem, I can not use tabby normally but use iterm2 or wezterm instead.

**Reply by Lockszmith-GH** [reply]
I really think this has to do with the way Tabby handles clipboard. Windows Terminal works flawlessly.
I recommend you follow the link I mentioned above and report this on Tabby's issue tracker
